filesystem/usr/local/share/landoop/etc/bashrc

```diff
@@ -2,3 +2,23 @@ export PS1="\[\033[1;31m\]\u\[\033[1;33m\]@\[\033[1;34m\]fast-data-dev \[\033[1;
 source /usr/share/bash-completion/bash_completion
 source /opt/landoop/tools/share/kafka-autocomplete/kafka
 alias ls='ls --color=auto'
+
+# set env variables and set defaults
+WEB_PORT=${WEB_PORT:-3030}
+ZK_PORT=${ZK_PORT:-2181}
+DEBUG=${DEBUG:-0}
+
+# check the status of zookeeper to provided $ZK_PORT
+ZK_STATUS="$(echo ruok | nc localhost $ZK_PORT | grep imok)"
+
+# Preconfigure lenses-cli
+for ((i=0;i<60;i++)); do
```

Comment:
Because bashrc loads on login, let's not make the user wait for zookeeper checks (also 99.9% of the time, ZK will be already up).